src/yxy.rs

```diff
@@ -60,7 +60,7 @@ impl<T: Float> Alpha<Yxy<T>, T> {
 
 impl<T: Float> FromColor<T> for Yxy<T> {
     fn from_xyz(xyz: Xyz<T>) -> Self {
-        let mut yxy = Yxy{ luma: xyz.y, ..Default::default() };
+        let mut yxy = Yxy{ x: T::zero(), y: T::zero(), luma: xyz.y };
```

Comment:
Oh, ok. Hmm... The Y=0 are is so strange, since it's mathematically possible to have "red, but no energy", but it makes no sense otherwise. We may just as well give in to the math and see where it takes us.